ci/pipeline

```diff
@@ -74,12 +76,13 @@ def checkLeakedProcesses(): Unit = {
 }
 
 @main
-def checkUnusedImports(logFileName: String): Unit = {
-  val logs = read! pwd / logFileName
-  val unusedImportMatch = """.*\.scala:\d+:\d+: Unused import.*\n.*import.*\n.*\^""".r
-  val unusedImports = unusedImportMatch.findAllIn(logs).toVector
-
-  assert(unusedImports.isEmpty, s"Found unused imports:\n${unusedImports.mkString("\n")}")
+def checkScalafix(): Unit = {
+  try {
+    %('sbt, "compile:scalafix --check")
+  } catch {
+    case e: InteractiveShelloutException =>
+      throw ScalafixException("Scalafix failed. Run sbt compile:scalafix before checking in.", e)
```

Comment:
This will print
```
[info] Running scalafix on 481 Scala sources
--- /Users/kjeschkies/src/marathon/src/main/scala/mesosphere/marathon/api/v2/validation/PodsValidation.scala
+++ <expected fix>
@@ -16,7 +16,6 @@
 import mesosphere.marathon.util.{RoleSettings, SemanticVersion}
 // scalastyle:on

-import akka.Done

 /**
   * Defines implicit validation for pods
[error] (Compile / scalafix) scalafix.sbt.ScalafixFailed: TestError
[error] Total time: 14 s, completed Mar 2, 2020 12:16:24 PM
ammonite.$file.ci.pipeline$ScalafixException: Scalafix failed. Run sbt compile:scalafix before checkin.
  ammonite.$file.ci.pipeline$.checkUnusedImports(pipeline:84)
  ammonite.$file.ci.pipeline$$routes$.$anonfun$apply$4(pipeline:419)
  ammonite.main.Router$EntryPoint.invoke(Router.scala:150)
  ammonite.main.Scripts$.runMainMethod(Scripts.scala:209)
  ammonite.main.Scripts$.$anonfun$runScript$5(Scripts.scala:144)
  ammonite.util.Util$.withContextClassloader(Util.scala:24)
  ammonite.main.Scripts$.$anonfun$runScript$4(Scripts.scala:108)
  ammonite.util.Res$Success.flatMap(Res.scala:62)
  ammonite.main.Scripts$.$anonfun$runScript$2(Scripts.scala:67)
  ammonite.util.Res$Success.flatMap(Res.scala:62)
  ammonite.main.Scripts$.$anonfun$runScript$1(Scripts.scala:46)
  ammonite.util.Res$Success.flatMap(Res.scala:62)
  ammonite.main.Scripts$.runScript(Scripts.scala:42)
  ammonite.Main.runScript(Main.scala:240)
  ammonite.MainRunner.$anonfun$runScript$1(Main.scala:403)
  ammonite.MainRunner.watchLoop(Main.scala:389)
  ammonite.MainRunner.runScript(Main.scala:403)
  ammonite.Main$.main0(Main.scala:320)
  ammonite.Main$.main(Main.scala:271)
  ammonite.Main.main(Main.scala)
ammonite.ops.InteractiveShelloutException
```